com.github.LongSoft.UEFITool.appdata.xml

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Please make sure to submit this upstream.

Reply:
https://github.com/LongSoft/UEFITool/pull/299